hal/targets/cmsis/TARGET_STM/TARGET_STM32F1/TARGET_BLUEPILL_F103C8/hal_tick.c

```diff
@@ -138,6 +138,21 @@ HAL_StatusTypeDef HAL_InitTick(uint32_t TickPriority) {
     return HAL_OK;
 }
 
+void HAL_SuspendTick(void)
+{
+    TimMasterHandle.Instance = TIM_MST;
+
+    // Disable HAL tick and us_ticker update interrupts (used for 32 bit counter)
+    __HAL_TIM_DISABLE_IT(&TimMasterHandle, (TIM_IT_CC2 | TIM_IT_UPDATE));
+}
+
+void HAL_ResumeTick(void)
+{
+    TimMasterHandle.Instance = TIM_MST;
+
+	// Enable HAL tick and us_ticker update interrupts (used for 32 bit counter)
```

Comment:
Can you fix the formatting of these 2 lines? I noticed them as well in some ST code...